### ISSUE TITLE
Braille font is very small and can’t be adjusted

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31762]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

For Low Vision users who are learning or teaching Braille music or transcribing the font of the Braille is very small.  This does not seem to scale with other scaling/magnification options or text settings.  

The panel itself can not be sized. Some users (possibly with older monitors)  report the edges of this panel are not visible at their top resolution available and it has been suggested this may also be impacting on Braille input that does not refresh as you type for some users. See [23176](https://github.com/musescore/MuseScore/issues/23176) 

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.5.2

### Regression

I was unable to check

### Operating system

iOS: Sequoia 15.6

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by jmwmusic** [comment]
I forgot to mention that the current Braille Panel displays 1 full line of Braille plus you can see the top bit of the next row.  As fully sighted users have also commented the font is rather small it might make sense to increase it enough to fit the vertical alignment with 1 full row of Braille only.
